components/search/MoreFiltersDropdown.vue

```diff
@@ -18,70 +18,85 @@
         {{ selectedOptionsCount }}
       </span>
     </template>
-    <b-dropdown-group class="more-facets-wrapper">
-      <b-dropdown-form v-if="showDateFilter">
+    <b-dropdown-form>
+      <li
+        class="more-facets-wrapper"
+        role="presentation"
+      >
+        <RadioGroupFilter
+          v-if="showApiToggle"
+          facet-name="api"
+          :options="['fulltext', 'metadata']"
+          :selected="preSelected['api'] || 'fulltext'"
+          @change="updateSelected"
+        />
         <DateFilter
+          v-if="showDateFilter"
           :name="PROXY_DCTERMS_ISSUED"
           :start="dateFilter.start"
           :end="dateFilter.end"
           :specific="dateFilter.specific"
           @dateFilter="dateFilterSelected"
         />
-      </b-dropdown-form>
-      <template
-        v-for="(facet, index) in moreFacets"
-      >
-        <MoreFiltersDropdownFacet
-          v-if="facet.fields && facet.fields.length > 0"
-          :key="index"
-          :fields="facet.fields"
-          :name="facet.name"
-          :selected="preSelected[facet.name]"
-          @selectedOptions="updateSelected"
-        />
-      </template>
-    </b-dropdown-group>
-    <li
-      class="dropdown-buttons"
-    >
-      <b-button
-        variant="link"
-        :disabled="!anyOptionsSelected"
-        data-qa="reset filter button"
-        @click="resetFilters"
-      >
-        {{ $t('facets.button.reset') }}
-      </b-button>
-      <b-button
-        variant="link"
-        data-qa="cancel button"
-        @click="cancelHandler"
-      >
-        {{ $t('facets.button.cancel') }}
-      </b-button>
-      <b-button
-        variant="primary"
-        :disabled="selectedOptionsUnchanged"
-        data-qa="apply button"
-        @click="applySelected"
+        <template
+          v-for="(facet, index) in moreFacets"
+        >
+          <MoreFiltersDropdownFacet
+            v-if="facet.fields && facet.fields.length > 0"
+            :key="index"
+            :fields="facet.fields"
+            :name="facet.name"
+            :selected="preSelected[facet.name]"
+            @selectedOptions="updateSelected"
+          />
+        </template>
+      </li>
+      <li
+        class="dropdown-buttons"
+        role="presentation"
       >
-        {{ $t('facets.button.apply') }}
-      </b-button>
-    </li>
+        <b-button
+          variant="link"
+          :disabled="!anyOptionsSelected"
+          data-qa="reset filter button"
+          @click="resetFilters"
+        >
+          {{ $t('facets.button.reset') }}
+        </b-button>
+        <b-button
+          variant="link"
+          data-qa="cancel button"
+          @click="cancelHandler"
+        >
+          {{ $t('facets.button.cancel') }}
+        </b-button>
+        <b-button
+          variant="primary"
+          :disabled="selectedOptionsUnchanged"
+          data-qa="apply button"
+          type="submit"
```

Comment:
The apply button has been made into a `submit` type button, not `button` type, so that hitting `Enter` will submit the form.